webapp/src/app/admin/admin.component.html

```diff
@@ -1,17 +1,9 @@
-<h1 class="text-3xl font-bold">Admin</h1>
-<div>
-  <div>User:</div>
-  {{ user() | json }}
-</div>
-
-@if (query.isPending()) {
-  <div>Loading...</div>
-} @else if (query.error()) {
-  <div>Something went wrong...</div>
-}
-@if (query.data(); as data) {
-  <div>
-    <div>Greeting:</div>
-    {{ data | json }}
+<!-- input for repositories with save button -->
+<h1 class="text-3xl font-bold mb-4">Environment</h1>
+<div class="flex flex-col items-start gap-4">
+  <div class="w-1/2">
+    <h2 class="text-lg font-semibold mb-2">Repositories to monitor:</h2>
+    <textarea hlmInput [formControl]="repositoriesForm" class="block size-fit" style="field-sizing: content"></textarea>
```

Comment:
I dont like the textfield. Its is small and not easy to use, maybe we could have the repositories in a list of textfields, where you can edit each

Reply:
Indeed, I should refactor this. At this point the PR is huge again, and there's so much stuff that could still be improved.